c/authService.c

```diff
@@ -33,8 +33,19 @@
 #include "socketmgmt.h"
 #include "zis/client.h"
 #include "httpserver.h"
+#include "zssLogging.h"
 
 #define SAF_CLASS "ZOWE"
+#define JSON_ERROR_BUFFER_SIZE 1024
+
+#define PASSWORD_RESET_RC_OK                0
+#define PASSWORD_RESET_RC_WRONG_PASSWORD    111
```

Comment:
Where do these values come from? Can you clarify that in a comment or in the names?